shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -446,7 +446,16 @@ namespace datadog::shared::nativeloader
 
                     moduleName = moduleName + EndLWStr;
                     instrumented_assembly_generator::WriteTextToFile(instrumented_assembly_generator::ModulesFileName, moduleName);
-                    instrumented_assembly_generator::CopyOriginalModuleForInstrumentationVerification(modulePath);
+                    if (instrumented_assembly_generator::IsCopyingOriginalsModulesEnabled())
+                    {
+                        // This option means we copy every loaded assembly to the InstrumentationVerification folder. It is off by default,
+                        // but can be useful as during an escalation, we can request
+                        //  that the customer turn it on, and then the customer
+                        // will only need to upload to the contents of the InstrumentationVerification folder to us for offline analysis, 
+                        // so we won't have to ask the customer to go and locate wherever it was that the assembly was loaded from at runtime.
+                        // In most cases, this would hopefully not be necessary as the output of the `dd-trace analyze-instrumentation` command should suffice.
```

Comment:
```suggestion
                        // In most cases, this would hopefully not be necessary as the textual output of the `dd-trace analyze-instrumentation` command should suffice.
```

Reply:
Fixed in 59dc3c9f22d7700ec54e60483dd11a150535c39c